datadog/resource_datadog_downtime.go

```diff
@@ -374,10 +374,9 @@ func updateDowntimeState(d *schema.ResourceData, dt *datadogV1.Downtime) error {
 	if err := d.Set("message", dt.GetMessage()); err != nil {
 		return err
 	}
-	if err := d.Set("monitor_id", dt.GetMonitorId()); err != nil {
-		return err
+	if v, ok := dt.GetMonitorIdOk(); ok && v != nil {
+		d.Set("monitor_id", v)
```

Comment:
I think we still want to check and report if an error gets returned from `d.Set`